test/post-deploy.js

```diff
@@ -22,13 +22,19 @@ const { expect } = chai;
 
 createTargets().forEach((target) => {
   describe(`Post-Deploy Tests (${target.title()})`, () => {
-    it('Purge a blog post', async () => {
-      await chai
+    it('invokes the function', async () => {
```

Comment:
As someone who has to implement a compatible version in a different environment, I would have hoped for better post-deploy tests that cover the different cases:
- single sheet origin, single sheet response
- multi-sheet origin, single sheet response
- multi-sheet origin, multi-sheet response
- single-sheet origin, sheet not found
- multi-sheet origin, sheet not found

Reply:
yes.... will come...